src/solc_0.5/Land.md

```diff
@@ -2,32 +2,146 @@
 
-The layerId is used to represent groups of lands called [Quads](https://en.wikipedia.org/wiki/Quadtree) we support quads
-of the following sizes: 24x24, 12x12, 6x6, 3x3. A user owns a quad if he owns all the lands inside it. Using quads a
-full group of lands can be transferred in one transaction. The contract also support batch transfers of quads and lands.
+- the coordinates `(x,y)` of the land inside the map.
+- the `layerId` that related to the size of [quads](#quads) (explained bellow).
 
-Land exists on ethereum (L1) and polygon (L2). There are two different contracts for each layer and also a land tunnel
-based on the matic fx-portal library.
+Land exists on ethereum (L1) and polygon (L2). There is a Land contract for each network and also a land tunnel based on
+the matic fx-portal library that can be used to transfer lands back and forth between the two networks.
 
-In the V1 of the land contract lands can only be minted on L1, V2 adds the possibility to mint lands on L2 too. V3
-implement the OpenSea royalties blacklist.
+In the V1 of the contract lands can only be minted on L1, V2 adds the possibility to mint on L2 too. V3 implement the
+OpenSea royalties blacklist.
+
+## Roles
 
 The land contract support the following roles:
 
-- admin: this is a unique address assigned during the initialization of the contract and can add/remove addresses from
-  the other roles.
-- minter: the addresses in this list can mint lands by calling the `mintQuad` method.
-- super operator: the addresses in this list are approved to transfer tokens between users.
-- meta transaction processor: the addresses in this can transfer lands in behalf of other users (similar to super
-  operators). On the L2 contract this role is the ERC2771 meta transaction forwarder.
+- admin: a unique address that is the owner of the contract and can manage the other roles.
+- minters: a list of addresses that can mint lands.
+- super operators: a list of addresses that are automatically approved to transfer tokens between users.
+- meta transaction processor: an address that can transfer lands in behalf of other users and used to implement
+  meta-transactions. On the L2 contract this role is the ERC2771 meta transaction forwarder.
+
+# Quads
+
+Quads are a way to groups lands. A quad of lands can be minted and transferred in one step. The concept is related to
+a [QuadTree](https://en.wikipedia.org/wiki/Quadtree) but it has it own specific implementation in the Land contract.
+
+The full map of `408x408` lands is divided in a grid of `17x17` quads that has a size of `24x24` each. Each `24x24`
+quad is divided in four `12x12` quads, each `12x12` quad is divided in four `6x6` quads, each `6x6`
+quad is divided in four `3x3` quads and each `3x3` quad is divided in nine `1x1` quads (or just lands). This way we have
+a QuadTree. Each node is a ***quad*** and has one ***parent-quad*** and some ***child-quads*** that he includes.
+
+![](images/TheGreatQuad.png)
```

Comment:
add a size, check the preview, it's huge 

https://www.markdownguide.org/hacks/#image-size

Reply:
We need that: "Markdown processor supports HTML" to use that. 
I can resize the images if you want ?